platform-sdk/swirlds-merkledb/src/main/java/com/swirlds/merkledb/MerkleDb.java

```diff
@@ -100,12 +101,9 @@ public final class MerkleDb {
     /** Label for database component used in logging, stats, etc. */
     public static final String MERKLEDB_COMPONENT = "merkledb";
 
-    /**
-     * Since {@code com.swirlds.platform.Browser} populates settings, and it is loaded before any
-     * application classes that might instantiate a data source, the {@link ConfigurationHolder}
-     * holder will have been configured by the time this static initializer runs.
-     */
-    private final MerkleDbConfig config;
+    /** Platform configuration */
+    @NonNull
+    private final Configuration configuration;
```

Comment:
It looks fine. However, I wonder if MerkleDb actually needs any other configs than `MerkleDbConfig`?

Reply:
Shall we resolve this one?